### ISSUE TITLE
Number of required epochs for fine tuning and some other questions

### ISSUE THREAD

**Issue by amirbakhtiary23** [lyuwenyu/RT-DETR#496]
Hi,
I used coco pre-trained coco weights to fine tune on VOD dataset with much fewer classes but the results after 12 epochs was rather disappointing. Therefore i have a few questions
1- So In general how many epochs do you suggest for a dataset of around 7K images? (data augmentation is  in order)
2- What loss value did you record in your experiments on coco and object365? 
3- How would one manage the label number assignment when fine tuning on a new dataset? 
Best regards,
